javascript/stimulus_reflex.js

```diff
@@ -480,6 +481,12 @@ const getReflexRoots = element => {
 const initialize = (application, initializeOptions = {}) => {
   const { controller, consumer, debug, params, isolate } = initializeOptions
   actionCableConsumer = consumer
+  setTimeout(() => {
+    if (Debug.enabled && consumer)
```

Comment:
I'm wondering if we should introduce another option to silence deprecation warnings? I have the feeling that deprecation warnings are not just "debug logs". What do you think?

Reply:
How would we transmit this to the client? Or do you mean a client option?

Given that you need to have debug on, don't we want them to see these?

Reply:
Yep a client option. `debug` is disabled by default so if you don't change anything you won't see the deprecation warnings.

I think an option like `deprecationWarnings` which is enabled by default so we can make sure the user really sees them. If he wants to ignore the warnings he can set `deprecationWarnings` to false. 

So it's not deepened on the `debug` option if the users sees the warnings or not.

Reply:
Would you go so far as to essentially duplicate the Debug mechanism? Or do you have something less intense in mind?

It seems like there are shortcuts we'd just regret if we push it down the line...

Reply:
Either to duplicate the mechanism we use in Debug or that we refactor the Debug stuff into a more general "config" construct so we can use it for multiple options. I don't have something less intense is mind.

But maybe it goes too far idk.

Reply:
I think I'm going to just duplicate the Debug mechanism you built. It works amazingly well, and we can leave it in the library for future deprecations.

Reply:
Ready for review!